checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitVisitor.java

```diff
@@ -348,7 +348,11 @@ public Void visitNewArray(NewArrayTree tree, Void p) {
                     componentType.getAnnotations(),
                     type.toString());
         }
-
+        List<? extends AnnotationMirror> annotations =
+                TreeUtils.annotationsFromArrayCreation(tree, 0);
+        if (AnnotationUtils.containsSame(annotations, NULLABLE)) {
+            checker.reportWarning(tree, "new.array.nullable.ignored");
+        }
```

Comment:
can we use this error message?
`new.array.type.invalid=annotations %s may not be applied as component type for array "%s"`

Reply:
No, because it is not about the component type.